packages/paste-core/components/slider/src/helpers.ts

```diff
@@ -0,0 +1,16 @@
+import type {UIStates, BackgroundColorTokens} from './types';
+
+export const getColorToken = (states: UIStates): BackgroundColorTokens => {
+  if (states.disabled) {
+    return 'colorBackgroundStrong';
+  } else if (states.error) {
+    return 'colorBackgroundError';
```

Comment:
question: Can't remember if you mentioned that there was no hover state on error, but I just noticed in chromatic 😬  Possible to add or nah? Sorry, didn't realize it wasn't in Figma.

If it is, we can use colorBackgroundErrorStrong like our other component error states.